docs/docs/users/general/creating-campaigns.md

```diff
@@ -0,0 +1,42 @@
+---
+id: creating-campaigns
+title: Creating and managing campaigns
+---
+Here is more information about campaigns. These can be either petition campaigns, event campaigns or fundraising campaigns. 
+
+## General info
+![](https://s3-eu-west-1.amazonaws.com/bureaubolster/Wings_2019-03-25_11-31-17.png)
```

Comment:
We should be able to bundle required images in the docs I think?

Reply:
Also, this screenshot is outdated. UI has changed.

Reply:
Bundling images in the docs is not yet a part of my iOS workflow. I can't easily save them and they would not show up when previewing the docs in iA Writer, because Working Copy needs to mirror an icloud folder. I can imagine we will find this neceassary in the future, but I need something that works *now*, in terms of something that lets me work efficiently and the manual site needs to go online ASAP.